File: C_Programming/Practice/program8.c
// Program to check whether number is even or odd

#include<stdio.h>
#include<stdbool.h>

bool CheckEvenOdd(int iNo)
{
    int iRemender = 0;

    iRemender = iNo % 2 ;

    if(iRemender == 0)
    {
        return true;
    }
    else
    {
        return false;
    }

}

int main()
{
    int iValue = 0;
    bool bRet = false;
    
    printf("Enter number to check whether it is even or odd : ");
    scanf("%d",&iValue);

    bRet = CheckEvenOdd(iValue);

    if(bRet == true)
    {
        printf("%d is even",iValue);
    }
    else
    {
        printf("%d is odd",iValue);
    }

    return 0;
}

File: Java_Programming/Practice/program8.java
// Check whether number is even or odd

import java.util.Scanner;

class program8
{
    public static int CheckEvenOdd(int iNo)
    {
        int iRemainder = 0;

        iRemainder = iNo % 2;

        return iRemainder;

    }

    public static void main(String A[])
    {
        int iValue = 0;
        int iRet = 0;
        
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter number : ");
        iValue = sc.nextInt();

        iRet = CheckEvenOdd(iValue);

        if(iRet == 0)
        {
            System.out.println("Number is Even");
        }

        else
        {
            System.out.println("Number is Odd");
        }

    }

}